src/payments/services/paymentEventsService.ts

```diff
@@ -241,6 +244,24 @@ export class PaymentEventsService {
     })
     await this.campaignsService.setIsPro(campaignId)
 
+    // For Segment
+    const sub = session.subscription as Stripe.Subscription
```

Comment:
```suggestion
    const subscription = session.subscription as Stripe.Subscription
```

Just makes it explicit what we're referring to